README_DEVELOPMENT.md

```diff
@@ -0,0 +1,142 @@
+# Using ovn-fake-multinode for development and testing
+
+## Setup the Vagrant environment
+
+This example explains how to use the ovn-fake-multinode environment when doing
+an Open vSwitch change, including a kernel module change. This assumes you are
+using the Vagrant environment.
+
+First we will checkout the ovn-fake-multinode environment as follows:
+
+```
+git clone https://github.com/ovn-org/ovn-fake-multinode.git
+cd ovn-fake-multinode
+export VM_CPUS=8
+export VM_MOUNT_NFS=true
+vagrant up && vagrant ssh
+```
+
+As there is no simple plugin to re-size the vagrant image with the libvirt
+provider, we will manually resize it. This will allow us to store and build
+a custom kernel, and meet the additional storage requirements for the
+updated docker images.
+
+The following set of commands will re-size the image:
+
+```
+vagrant halt
+qemu-img resize ~/.local/share/libvirt/images/ovn-fake-<XXX>.img +100G
+vagrant up && vagrant ssh
+echo ", +" | sudo sfdisk -N 1 /dev/vda --no-reread
+sudo partprobe
+sudo sudo xfs_growfs /
+exit
+vagrant reload && vagrant ssh && vagrant ssh
```

Comment:
There is "vagrant ssh" twice. Is it intentional ?

Reply:
Nope, will fix in next commit